README.md

```diff
@@ -1,14 +1,60 @@
+
+### Appearance
+The appearance of the .NET MAUI Barcode Generator is easily customizable. Customize barcode color, bar height, bar width, and more.
+
+### Color
+Apply a foreground color and background color to a barcode.
+
+### Bar height
+Specify the height of the barcode container or it will take the full container height.
+
+### Bar width
+Define the width of the smallest bar line or dot using the module property. This is not set: the size of the smallest bar line is determined depending on the width available.
+
+## Related Links
+[Learn More about .NET MAUI Barcode Generator](https://www.syncfusion.com/maui-controls/maui-barcodes?utm_source=github&utm_medium=listing&utm_campaign=maui-barcode-generator-github-samples)
+
+[Download Free Trial](https://www.syncfusion.com/downloads/maui?utm_source=github&utm_medium=listing&utm_campaign=maui-barcode-generator-github-samples)
+
+[Pricing](https://www.syncfusion.com/sales/teamlicense?utm_source=github&utm_medium=listing&utm_campaign=maui-barcode-generator-github-samples)
+
+[Documentation](https://help.syncfusion.com/maui/barcode-generator/getting-started?utm_source=github&utm_medium=listing&utm_campaign=maui-barcode-generator-github-samples)
+
+[View Demos](https://github.com/syncfusion/maui-demos/tree/master/MAUI/Barcode?utm_source=github&utm_medium=listing&utm_campaign=maui-barcode-generator-github-samples)
+
+[Community Forums](https://www.syncfusion.com/forums/maui?utm_source=github&utm_medium=listing&utm_campaign=maui-barcode-generator-github-samples)
+
+[Knowledge base](https://support.syncfusion.com/kb?utm_source=github&utm_medium=listing&utm_campaign=maui-barcode-generator-github-samples)
+
+[Suggest a feature or report a bug](https://www.syncfusion.com/feedback/maui?utm_source=github&utm_medium=listing&utm_campaign=maui-barcode-generator-github-samples)
+
+## About Syncfusion .NET MAUI Controls
+Syncfusion's [.NET MAUI Controls](https://www.syncfusion.com/maui-controls?utm_source=github&utm_medium=listing&utm_campaign=maui-barcode-generator-github-samples) library is the only suite that you will ever need to build an application since it contains over 40 high-performance, lightweight, modular, and responsive UI controls in a single package. In addition to ListView, we provide popular .NET MAUI Controls such as [DataGrid](https://www.syncfusion.com/maui-controls/maui-datagrid?utm_source=github&utm_medium=listing&utm_campaign=maui-barcode-generator-github-samples), [Charts](https://www.syncfusion.com/maui-controls/maui-cartesian-charts?utm_source=github&utm_medium=listing&utm_campaign=maui-barcode-generator-github-samples), [Scheduler](https://www.syncfusion.com/maui-controls/maui-scheduler?utm_source=github&utm_medium=listing&utm_campaign=maui-barcode-generator-github-samples), and [Excel](https://www.syncfusion.com/document-processing/excel-framework/maui?utm_source=github&utm_medium=listing&utm_campaign=maui-barcode-generator-github-samples).
+

```

Comment:
Recheck on this
![image](https://github.com/SyncfusionExamples/getting-started-with-the-dotnet-maui-barcode-generator/assets/95272441/02873715-79bc-4ddd-aacc-26eb2a7fbb6c)

Reply:
Updated the correct control name.